mavros/include/mavros/mavros_uas.h

```diff
@@ -416,43 +417,182 @@ class UAS {
 	}
 
 	/**
-	 * @brief Transform frame between ROS and FCU. (Vector3d)
+	 * @brief Orientation transform options when applying rotations to data
+	 */
+	enum class STATIC_TRANSFORM: uint8_t {
+		NED_TO_ENU, //! will change orinetation from being expressed WRT NED frame to WRT ENU frame
+		ENU_TO_NED,  //! change from expressed WRT ENU frame to WRT NED frame
+		AIRCRAFT_TO_BASELINK, //! change from expressed WRT aircraft frame to WRT to baselink frame
+		BASELINK_TO_AIRCRAFT //! change from expressed WRT baselnk to WRT aircraft
+	};
+
+	/**
+	 * @brief Transform representation of attitude from 1 frame to another
+	 * (e.g. transfrom attitude from representing  from base_link -> NED
+	 		 to representing base_link -> ENU)
+
+	 * General function. Please use specialized enu-ned and ned-enu variants.
```

Comment:
Add doxygen tags to each of the inline functions?  Even though they're missing now